preprod.healdata.org/metadata/aggregate_config.json

```diff
@@ -529,8 +529,572 @@
 					"authz": "/programs/open",
 					"study_url": "https://pdaps.org/datasets/laws-regulating-administration-of-naloxone-1501695139",
 					"dataset_1_title": "Naloxone_Data_09112020.xlsx, 20180810_Good_Samaritan_Law_Stat_Data.xlsx, 20170216-RM-Stat-Data.xlsx"
-				}
-			}
-		}
-	}
-}
+				},
+                "57b45d83d6c9e7e8693ccdfb": {
```

Comment:
Could you remind me again where these numbers are coming from? from PDAPS DB?

Reply:
Yeah - these numbers are coming from the PDAPS DB 
`def get_pdaps_id(dataset_name):
    #mds_url and study_id from config file
    mds_url = "https://api.monqcle.com"
    study_id = dataset_name
    url = f"{mds_url}/siteitem/{study_id}/get_by_dataset?site_key=56e805b9d6c9e75c1ac8cb12"
    response = requests.get(url)
    #extract the display id as the per item key
    api_call = json.loads(response.text)
    #here we only need the id for the key
    return api_call['id']`